src/task_4_3.py

```diff
@@ -0,0 +1,18 @@
+#  Дан словарь: {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
+# Добавить каждому ключу число равное длине этого ключа (пример {‘key’: ‘value’} -> {‘key3’:
+# ‘value’}). Чтобы получить список ключей - использовать метод .keys()
+a = {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
+n = 0
+while n < 4:
+    key = list(a.keys())
+    y = key.pop(n)
+    z = str(len(y))
+    s = str(y) + z
+    a[s] = a.pop(y)
+    n += 1
+print(a)
+
+dct = {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
+for key in list(dct.keys()):
```

Comment:
```suggestion
for key, value in dct.items():
```
https://docs-python.ru/tutorial/operatsii-slovarjami-dict-python/metod-dict-items/#:~:text=%D0%9C%D0%B5%D1%82%D0%BE%D0%B4%20dict.items()%20%D0%B2%D0%BE%D0%B7%D0%B2%D1%80%D0%B0%D1%89%D0%B0%D0%B5%D1%82%20%D0%BD%D0%BE%D0%B2%D1%8B%D0%B9%20%D1%81%D0%BF%D0%B8%D1%81%D0%BE%D0%BA%2D%D0%BF%D1%80%D0%B5%D0%B4%D1%81%D1%82%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5,value)%2C%20%D1%81%D0%BE%D1%81%D1%82%D0%BE%D1%8F%D1%89%D0%B8%D0%B9%20%D0%B8%D0%B7%20%D1%8D%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2%20%D1%81%D0%BB%D0%BE%D0%B2%D0%B0%D1%80%D1%8F